### PR TITLE
Adding a cask for Grisbi

### PR DESCRIPTION
**Pull request by frantzmiccoli**
I don't know if I was suppose to present it checking boxes in the template but here it is ;)
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/grisbi.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'grisbi' do
+  version '1.0.0'
+  sha256 'c7a84e8737468e3c1dc08dec5a34330158c767b3ef5a64716d6ec3a0b2c879c2'
+
+  url 'http://downloads.sourceforge.net/project/grisbi/grisbi%20stable/1.0.x/Grisbi-1.0.0.dmg?r=&ts=1460142325&use_mirror=heanet'
```
Comment (vitorgalvao) [inline Casks/grisbi.rb]: Doesn’t `http://downloads.sourceforge.net/project/grisbi/grisbi%20stable/1.0.x/Grisbi-1.0.0.dmg` suffice?

Comment (frantzmiccoli) [inline Casks/grisbi.rb]: No, this lead to the "waiting for your download" page on SourceForge.

Comment (vitorgalvao) [inline Casks/grisbi.rb]: > No, this lead to the "waiting for your download" page on SourceForge.

And then it actually performs the download.

Many casks use that URL. `curl` has no issue with it. Please try it on the cask.

**Comment by frantzmiccoli** [comment]
@vitorgalvao Done, you were right.

File: Casks/grisbi.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'grisbi' do
+  version '1.0.0'
+  sha256 'c7a84e8737468e3c1dc08dec5a34330158c767b3ef5a64716d6ec3a0b2c879c2'
+
+  # http://downloads.sourceforge.net/project/grisbi/grisbi%20stable/1.0.x/Grisbi-1.0.0.dmg was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/grisbi.rb]: Longer than needed. As it is, we would have to change it every time.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in f6a7fcf74b3c198612d371602fa5b125ef4d9f38. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**